webapp/app/forms/steps/lotse/fahrkostenpauschale.py

```diff
@@ -0,0 +1,149 @@
+
+
+from flask import render_template
+from flask_babel import lazy_gettext as _l, ngettext, _
+from wtforms.validators import InputRequired
+from wtforms import SelectField
+from flask_wtf.csrf import generate_csrf
+
+from app.model.components import FahrkostenpauschaleProps
+from app.model.components.helpers import form_fields_dict
+from app.forms import SteuerlotseBaseForm
+from app.forms.steps.step import SectionLink
+from app.forms.steps.lotse.lotse_step import LotseFormSteuerlotseStep
+from app.forms.steps.lotse.personal_data import PersonAHasDisabilityPrecondition, PersonBHasDisabilityPrecondition, ShowPersonBPrecondition, StepFamilienstand, get_number_of_users
+
+
+def calculate_fahrkostenpauschbetrag(has_pflegegrad=False, disability_degree=None, has_merkzeichen_bl=False, has_merkzeichen_tbl=False,
+                                     has_merkzeichen_h=False, has_merkzeichen_ag=False, has_merkzeichen_g=False):
+    if has_pflegegrad or has_merkzeichen_bl or has_merkzeichen_tbl or has_merkzeichen_tbl or has_merkzeichen_h or has_merkzeichen_ag:
+        return 4500
+    elif disability_degree is not None and disability_degree >= 80 or (has_merkzeichen_g and disability_degree >= 70):
+        return 900
+
+    return 0
+
+
+class StepFahrkostenpauschale(LotseFormSteuerlotseStep):
+
+    def get_overview_value_representation(self, value):
+        result = None
+
+        if value == 'yes':
+            result = str(self.get_fahrkostenpauschale()) + ' ' + _('currency.euro')
```

Comment:
Shouldn't we add an abstract `get_fahrkostenpauschale` method? If we do so, we should probably add one to the PauschbetragStep, too.

Reply:
:+1: